src/actions/clientActions.js

```diff
@@ -0,0 +1,22 @@
+import { CLIENT_CHANGE } from '../constants/actionTypes'
+
+function isSmallScreen () {
+  return global.window.innerWidth <= 768
+}
+
+function clientChange (smallScreen) {
+  return {
+    type: CLIENT_CHANGE,
```

Comment:
So with “client” you mean a web browser, so technically correctly User Agent (UA). ;)